templates/partials/button.html

```diff
@@ -0,0 +1,90 @@
+{% set disabled = disabled_status|default('') %}
+
+{% set button_library = {
+        "reg_action_button": {
+            "class": "button-reg d-button-text"
+        },
+        "reg_action_button_alt": {
+            "class": "button-reg d-button-text"
+        },
+        "reg_saving_button": {
+            "class": "button-reg d-button-text",
+            "animated_spinner": '<i id="spinnerAnimated" class="fa-thin fa-circle-notch fa-spin visibility-spinner"></i>',
```

Comment:
spinner does not appear without the `fa-solid` class as well

Reply:
RIght.  I missed than the fa-thin version is for the pro version of fontawesome.

Class changed to fa-solid.